content/zh-cn/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -183,6 +183,18 @@ container. This co-location allows them to interact closely and share resources.
 
 边车容器与主容器共享相同的网络和存储命名空间。这种共存使它们能够紧密交互并共享资源。
 
+<!--
+From Kubernetes perspective, sidecars graceful termination is less important.
+When other containers took all alloted graceful termination time, sidecar containers
+will receive the `SIGTERM` following with `SIGKILL` faster than may be expected. 
+So exit codes different from `0` (`0` indicates successful exit), for sidecar containers are normal
+on Pod termination and should be generally ignored by the external tooling.
+-->
+从 Kubernetes 的角度来看，Sidecar 容器的优雅终止（Graceful Termination）相对不那么重要。
```

Comment:
```suggestion
从 Kubernetes 的角度来看，边车容器的优雅终止（Graceful Termination）相对不那么重要。
```

Reply:
我仍然觉得，容器的终止，无论是自杀还是他杀，都没有“优雅”二字可言。
我并不在乎其他媒体如何翻译，只是坚持要忠于英文原文所隐含的意思。
Grace 在剑桥辞典中有若干含义：
1. 优美、优雅、雍容：如果采用这个含义，我们可以认为容器的终止是smooth的、relaxed，可以用“雍容地、平静地死去”来理解；
2. 体面、有礼貌、公平：如果采用这个含义，我们可以理解所强调的是容器死的并不难看，很从容，有一定的尊严。
3. 恩典、恩宠：带有一些上帝怜悯终生的角度。如果采用这个语义，那容器死掉时可能有一些感恩涕零的味道。似乎也不太对。
4. 宽限期、延缓期：这个我觉得是最接近原来含义的。Grace 作不可数名词是可以理解为宽限期。英文解释是 a  period of time left or allowed before something happens or before something must be done. 例句：The exams have been postponed, so the students have a few day's grace before they start. 如果采用这个含义，那么 graceful 可以翻译为“带宽限期的”。

综上，第四种最接近原意，但据我所知英汉辞典中都这样翻译过。比较接近的翻译，尤其是在描述一件并不那么美好的事物时，
大多选择第2种词义。形容过程平稳，留有一定的尊严。例如：

Browser support for some is limited, but CSS is designed for graceful degradation; a page should still be legible and usable if a style isn't applied.

浏览器对某些选择符的支持还不是很充分，但是CSS的设计允许平稳退化；如果无法应用某个样式，页面应该仍然是合法有效的。

For Mr Semel, who is 64, a sale might also hold the appeal of a graceful exit. It would be a rueful one, however.

对64岁的塞梅尔先生而言，售卖交易也可能保持体面出局的风度，但他会心怀悔意。

无论如何，“优雅”二字是主体内生的一种表现，而不是外界所能够营造的。这里的 graceful termination，
并不是容器所能主导。通俗地讲，容器可能想要优雅，但是环境未必允许。环境能够允许它平稳地、体面地结束，
已经是一种让步。